packages/sitecore-jss-react/src/components/Image.tsx

```diff
@@ -99,6 +99,19 @@ const getImageAttrs = (
     return null;
   }
 
+  // we want to get rid of class prop in compliance with JSX
```

Comment:
This is an EE-only issue, correct? If so, can we do this inside the [EE-specific code block](https://github.com/Sitecore/jss/blob/dev/packages/sitecore-jss-react/src/components/Image.tsx#L148-L155)? In another branch, we have this code [exported to be re-used in a Next.js image component](https://github.com/Sitecore/jss/blob/dev/packages/sitecore-jss-react/src/components/Image.tsx#L148-L155). This would make it easier to merge this fix into this branch.

Reply:
Bug is EE-specific, but LayoutService can pass image JSON data with "class" param in every scenario. Whatever img tag is rendered by the renderField pipeline will be transformed into JSON without modifications. So it is important to do this check in all cases, not just EE.

Reply:
Hmm. In normal mode, I don't know how the image field would have a 'class' prop in the JSON data from Layout Service, especially in the case the field is empty.

Also, whoops, I had incorrect URL for second link. Should be this: [exported to be re-used in a Next.js image component](https://github.com/Sitecore/jss/blob/feature/449842-nextjs-image/packages/sitecore-jss-react/src/components/Image.tsx#L124)

Reply:
As discussed, let's stick with this, and I will add a note to account for this in the next/image component.